### PR TITLE
[tls handshaked] fix:Interrupted system call (os error 4)

### PR DESCRIPTION
**Pull request by randsoy**
Dynamically load the FFI Library using flutter in Android system.  `Connection Failed: tls connection init failed: Interrupted system call (os error 4)`(ureq error);

### CODE REVIEW

File: rustls/src/conn.rs [hunk 1]
```diff
@@ -478,11 +478,27 @@ impl<Data> ConnectionCommon<Data> {
...
+                debug!("read_tls");
+                match self.read_tls(io) {
+                    Ok(0) => {
+                        eof = true;
+                        debug!("eof");
+                        break;
+                    }
+                    Ok(n) => {
+                        rdlen += n;
+                        debug!("read {} bytes", n);
```
Comment (djc) [inline rustls/src/conn.rs:491]: I think we should remove the `debug!()` call here, as that seems pretty noisy and not that useful?

File: rustls/src/conn.rs [hunk 2]
```diff
@@ -478,11 +478,27 @@ impl<Data> ConnectionCommon<Data> {
...
+                        break;
+                    }
+                    Err(ref err) if err.kind() == io::ErrorKind::Interrupted => {
+                        // nothing to do
+                        debug!("complete_io: read interrupted");
+                    }
+                    Err(err) => {
```
Comment (djc) [inline rustls/src/conn.rs:498]: Style nit: could use `return` as an expression here (omitting brances and semi-colon here).

**Review by djc — COMMENTED** [review]
Thanks for your contribution, I think this makes sense.

Could you change the first line of the commit message to describe the intended change, please? Also commented on some nits below.

**Comment by djc** [comment]
We'd like to publish 0.20.5 soon. Would be great if you can fix up the formatting issues!

**Comment by randsoy** [comment]
> We'd like to publish 0.20.5 soon. Would be great if you can fix up the formatting issues!

` cargo fmt`

**Review by djc — APPROVED** [review]

**Pull request merged**